lib/resources/example_json/r4_examples_service_request.rb

```diff
@@ -42,6 +42,11 @@ module Resources
         "priority": "routine",
         "code": {
           "coding": [{
+            "system": "https://fhir.cerner.com/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/codeSet/200",
```

Comment:
I think we've got an effort to go through the resources and update the tenant to our new sandbox. not sure if we need to do that here.

Reply:
The effort is currently being made through: https://github.com/cerner/fhir.cerner.com/pull/420/.
The assumption is we would like to switch to C1941 in a single go and so no updates are going to be made at this point in time. Thanks!